### ISSUE TITLE
scide: syntax colors affects ide colors

### ISSUE THREAD

**Issue by htor** [supercollider/supercollider#4059]
when selecting a color theme in preferences you also change the colors for the entire gui.
users may not want to have the entire ide in the same color theme as their syntax highlighting theme, because it may reduce the contrast/usability or make it look awkward (see screenshot number 1). 

i suggest we add "ui theme" as a second option under preferences that let's users pick themes for tabs, window borders, window bg and fg colors etc.. in the same way that color themes works now. 
this makes sense, since highlighting code and styling the ui are two different problems to solve. 

alternatively, we fix the styling on these ui elements, without the possibility of customization (e.g. use gray colors for ui) so that the ui is consistent and usable across all syntax themes regardless of their look & feel. 

screenshot 1
<img width="1280" alt="screen shot 2018-09-13 at 09 08 10" src="https://user-images.githubusercontent.com/1709263/45473058-37f94480-b736-11e8-88a0-46cc18653a84.png">

this shows how the atom editor renders it with a light syntax theme and a dark ui theme:
<img width="1280" alt="screen shot 2018-09-13 at 09 32 41" src="https://user-images.githubusercontent.com/1709263/45473687-008b9780-b738-11e8-871c-f672eae0d5d1.png">

Environment
-----------

* Your SuperCollider version: SuperCollider 3.10.0-beta2 [c28e53b9]

* Your operating system and version: macos 10.13

**Comment by snappizz** [comment]
yeah there should be a way to do more fine customization of the editor and IDE themes. seems like kind of a big project, and hard to prioritize right now -- but if you want to tackle this, i'd be happy to help you if needed.

one thing we can address in the sort term: i do think we should leave the preferences editor alone in the default fusion theme. you're right, it really does look bad for low-contrast themes like solarized light.

**Comment by brianlheim** [comment]
I am working on this today. IIUC all I am going to do is make it so the preferences window has default coloring. All other UI elements, including docklets, main editor window, and other popups like "look up implementations", will remain the same. Then, I suppose after that we should add a way to make the UI settings configurable. We can leave this issue open and rename it to reflect that.

IIUC, this change was exactly what was desired in #3950, and that was generally regarded as an improvement (rightfully so IMO). So I don't want to just revert that.
